tests/bearlib/aspects/CollectionsTest.py

```diff
@@ -1,6 +1,6 @@
 import pytest
 
-from coalib.bearlib.aspects import aspectTypeError
+from coalib.bearlib.aspects import ApectTypeError
```

Comment:
... OR ... use `from coalib.bearlib.aspects import \n    (ApectTypeError as aspectTypeError)` so you do not need to modify the `pytest.raises`